utils/config/daos_agent.yml

```diff
@@ -99,7 +99,11 @@
 #    iface: ib3
 #    domain: mlx5_3
 
-# Manually force a specific fabric provider to be used by all clients, in the event that the server
-# supports multiple providers.
+## Select a secondary fabric provider to be used by all clients, if the server is configured with
+## multiple providers. The provider index is 0-based and based on the order of fabric providers
+## listed in the "provider" field of the corresponding daos_server configuration. Index 0
+## corresponds to the primary provider.
 #
-#provider: ofi+verbs
+## default: 0
+#
+#provider_idx: 1
```

Comment:
this is confusing to me, shouldn't this be called secondary_provider_idx according to the text above? and, just to clarify, if this gets set to 0 then no secondary provider is set (default)?

Reply:
yes looks good to me